platform/viewer/src/appInit.js

```diff
@@ -49,6 +50,12 @@ function appInit(appConfigOrFunc, defaultExtensions) {
     appConfig,
   });
 
+  if (window) {
```

Comment:
why checking `window`?

Reply:
I could just avoid that, and have the check inside the ConfigPoint load itself - it is just that it is only relevant if you are actually in a browser and have a URL that you can fetch the data from.  If you want, I will move the check into config point itself.